install_config/oab_broker_configuration.adoc

```diff
@@ -740,3 +740,112 @@ secrets:
   secret: db_creds
   apb_name: dh-rhscl-postgresql-apb
 ----
+
+[[configuring-oab-proxy]]
+== Running Behind a Proxy
+
+When running the OAB inside of a proxied {product-title} cluster, it is
+important to understand its core concepts and consider them within the context
+of a proxy used for external network access.
+
+As an overview, the broker itself runs as a pod within the cluster. It has a
+requirement for external network access depending on how its registries have
+been configured.
+
+[[configuring-oab-proxy-prerequisites]]
+=== Prerequisites
+
+Some important prerequisites necessary for running the broker behind a proxy:
+
+[[configuring-oab-proxy-cluster-proxy]]
+==== Configuring the Cluster to Run Behind a Proxy
+
+Before attempting to run the OAB behind a proxy, review
+xref:../install_config/http_proxies.adoc#install-config-http-proxies[Working
+with HTTP Proxies] and ensure your cluster is configured accordingly to run
+behind a proxy.
+
+In particular, the cluster must be configured to _not_ proxy internal cluster
+requests. This is typically configured with a `NO_PROXY` setting of
+`.cluster.local,.svc`, in addition to any other desired `NO_PROXY` settings.
+This is because the broker must be able to directly communicate with its etcd
```

Comment:
Your doc should contain PR https://github.com/openshift/ansible-service-broker/pull/806/files